src/tools/build-manifest/src/main.rs

```diff
@@ -171,7 +170,12 @@ impl Builder {
         self.cargo_version = self.version("cargo", "x86_64-unknown-linux-gnu");
 
         self.digest_and_sign();
-        let manifest = self.build_manifest();
+        let Manifest { manifest_version, date, pkg } = self.build_manifest();
+        let mut manifest = BTreeMap::new();
+        manifest.insert("manifest-version".to_string(),
```

Comment:
Please add a comment explaining that rebuilding of the manifest BTreeMap is to correct the 'manifest-version' name.